sdks/python/apache_beam/coders/coder_impl.pxd

```diff
@@ -69,8 +69,9 @@ cdef class DeterministicFastPrimitivesCoderImpl(CoderImpl):
 
 
 cdef object NoneType
-cdef char UNKNOWN_TYPE, NONE_TYPE, INT_TYPE, FLOAT_TYPE, BOOL_TYPE
-cdef char BYTES_TYPE, UNICODE_TYPE, LIST_TYPE, TUPLE_TYPE, DICT_TYPE, SET_TYPE
+cdef unsigned char UNKNOWN_TYPE, NONE_TYPE, INT_TYPE, FLOAT_TYPE, BOOL_TYPE
```

Comment:
Curious, what was a motivation for this?

Reply:
Previously the decoder put the "decode via fallback" in an else clause. I tightened this up to require the tag to be UNKNOWN_TYPE throwing an exception if an invalid tag was given, but this exposed the fact that 0xFF upcasts to int differently depending on whether char is signed or unsigned. This make it more explicit and consistent with stream.read_byte().